dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -10,6 +10,7 @@
 - hide_source ||= false
 - use_phaser ||= false
 - use_makerlab ||= false
+- app_options[:locale] = js_locale
```

Comment:
it's not clear to me why this is the best place to set this.

Can we also remove the now-defuct section from _blockly.html.haml?